hibike/travis/Makefile

```diff
@@ -12,7 +12,7 @@ artifacts-install:
 	$(nop)
 
 lint:
-	cd .. && pylint *.py
+	cd .. && find | grep "\\.py$$" | xargs pylint
```

Comment:
No, that only works for files one directory down. I would like to lint all files in the directory tree. There might be a way to remove the `grep`.